### ISSUE TITLE
 Get the code-demos cross-refs right 

### ISSUE THREAD

**Issue by tpielok** [compstat-lmu/lecture_i2ml#355]
There are code-demos which are cross-referencing each other.
Make sure these still work once the new structure of the course website etc is fixed.

**Comment by tpielok** [comment]
Shall we also upload the rendered code demos? @fabian-s 
Then we could just link to the code demos on the master branch, e.g. 
`https://github.com/compstat-lmu/lecture_i2ml/tree/master/code-demos-pdf/code_demo_limo.pdf`.
They could also be used on the course page. (At the moment it only links to the Rmd files. Also, I think we should state there, that some code demos depend on each other, s.t. one should download all to the same folder, but this is probably a different issue...?)

**Comment by fabian-s** [comment]
> Shall we also upload the rendered code demos? @fabian-s
> Then we could just link to the code demos on the master branch, e.g.
> `https://github.com/compstat-lmu/lecture_i2ml/tree/master/code-demos-pdf/code_demo_limo.pdf`.
> They could also be used on the course page. 

Yeah, that makes a lot of sense -- plz add something to the Makefile to automatically copy them to 
a new subfolder /code-demos-pdf 

> (At the moment it only links to the Rmd files. Also, I think we should state there, that some code demos depend on each other, s.t. one should download all to the same folder, but this is probably a different issue...?)
aah. yeah. good point, too. see https://github.com/compstat-lmu/lecture_i2ml/commit/3f41c5b692714e18f95805ef7a9fa632a69375b1
